### DISCUSSION TITLE
Classic CSOM to PnP.Core

### DISCUSSION THREAD

**Discussion by PrasadMadhavan** [pnp/pnpcore#827] [Q&A] [answered]
I am upgrading one of our SharePoint solutions (Backend Jobs) from .Net Framework 4.6 to .Net Core. As a part of this activity, I am trying to replace the existing CSOM code with PnP.Core.

A part of the classic CSOM code looks like this.
            clientContext.Web.LoadAsync(clientContext.Web.Lists,
                lists => lists.include(
                    l => l.Title,
                    l => l.EventReceivers).Where
                        (l => l.Title == listName));

I am trying to replace this code in PnP.Core as shown below. But it doesn't recognize EventReceivers property.  

          await clientContext.Web.Lists.LoadAsync(l => l.Id, l => l.Title, l => l.EventReceivers);
            var lists = await clientContext.Web.Lists.Where(l => l.Title == listName).ToListAsync();

Can you please help? Can we access remote event receivers in code using PnP.Core?

**Comment by jansenbe** [comment, marked as answer]
@PrasadMadhavan : Followed up via https://github.com/pnp/pnpcore/issues/828.
